test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/NuGet.PackageManagement.UI.Test.csproj

```diff
@@ -63,6 +63,9 @@
   </ItemGroup>
   <ItemGroup>
     <EmbeddedResource Include="compiler\resources\EntityFrameworkSearch.json" />
+    <EmbeddedResource Include="compiler\resources\index.json" />
```

Comment:
same as the above problem.

Reply:
Rebased target feature branch. Refreshed the base on this page.